LoopUI/Views/SettingsView.swift

```diff
@@ -0,0 +1,207 @@
+                               image: Image(uiImage: viewModel.cgmManagerSettingsViewModel.image, bundleString: "Dexcom G6"),
+                               label: viewModel.cgmManagerSettingsViewModel.name,
+                               details: viewModel.cgmManagerSettingsViewModel.details)
+        } else {
+            // TODO: this "dismiss then call onTapped()" here is temporary, until we've completely gotten rid of SettingsTableViewController
+            return LargeButton(action: { self.dismiss(); self.viewModel.cgmManagerSettingsViewModel.onTapped() },
+                               image: Image(bundleString: "Dexcom G6"),
+                               label: LocalizedString("Add CGM", comment: "Title text for button to add CGM device"),
+                               details: LocalizedString("Tap here to set up a CGM", comment: "Descriptive text for button to add CGM device"))
+        }
+    }
+    
+    private var supportSection: some View {
+        Section (header: SectionHeader(label: NSLocalizedString("Support", comment: "The title of the support section in settings"))) {
+            NavigationLink(destination: Text("Support")) {
+                Text(NSLocalizedString("Support", comment: "The title of the support section in settings"))
+            }
+        }
+    }
+
+}
+
+fileprivate struct LargeButton: View {
+    
+    let action: () -> Void
+    var includeArrow: Bool = true
+    let image: Image
+    let label: String
+    let details: String
+
+    // TODO: The design doesn't show this, but do we need to consider different values here for different size classes?
+    static let spacing: CGFloat = 15
+    static let imageWidth: CGFloat = 48
+    static let imageHeight: CGFloat = 48
+    static let topBottomPadding: CGFloat = 20
+    
+    public var body: some View {
+        Button(action: action) {
+            HStack {
+                HStack(spacing: Self.spacing) {
+                    image
+                        .renderingMode(.original)
+                        .resizable()
+                        .scaledToFit()
+                        .frame(width: Self.imageWidth, height: Self.imageHeight)
+                    VStack(alignment: .leading) {
+                        Text(label)
+                            .foregroundColor(.primary)
+                        DescriptiveText(label: details)
+                    }
+                }
+                if includeArrow {
+                    Spacer()
+                    // TODO: Ick. I can't use a NavigationLink because we're not Navigating, but this seems worse somehow.
+                    Image(systemName: "chevron.right").foregroundColor(.gray).font(.footnote)
+                }
+            }
+            .padding(EdgeInsets(top: Self.topBottomPadding, leading: 0, bottom: Self.topBottomPadding, trailing: 0))
+        }
+    }
+}
+
+extension Image {
+    
+    init(uiImage: UIImage? = nil, bundleString: String) {
```

Comment:
It isn't obvious to me from this function signature that the behavior is 'use the UIImage if present, or fallback on a framework image if not'.

Consider something like `init(uiImage: UIImage?, orFallback bundleString: String)`